I made some tweaks...

afrojer opened the pull request:
Pull in shell customizations from a directory (default to ~/.env.d)
Add some default customizations like my find functions
Break out shell color processing into a separate file (could possibly
search the customization directory first...)

nviennot committed:
Changed git_prompt_info() and the gentoo theme

nviennot committed:
Made the histfile bigger

nviennot committed:
Added own .zshrc

nviennot committed:
README

nviennot committed:
Added noh, rvm support, hitch support

nviennot committed:
Changed gentoo theme

nviennot committed:
Added an alias nogit to disable the git prompt (heavy repo makes it slow)

nviennot committed:
Removed the nomatch option

nviennot committed:
Tweaked the PATH

nviennot committed:
RVM fix

nviennot committed:
Merge remote-tracking branch 'upstream/master'

nviennot committed:
Merge remote-tracking branch 'upstream/master'

Conflicts:
	README.textile

nviennot committed:
Added the zsh-syntax-highlighting plugin

nviennot committed:
Updated readme

Commit:
added some flexibility to the environment

Pull in shell customizations from a directory (default to ~/.env.d)
Add some default customizations like my find functions
Break out shell color processing into a separate file (could possibly
search the customization directory first...)

nviennot commented:
wonderful, will look this afternoon :D

On Thursday, September 22, 2011 06:42:56 you wrote:

> Pull in shell customizations from a directory (default to ~/.env.d)
> Add some default customizations like my find functions
> Break out shell color processing into a separate file (could possibly
> search the customization directory first...)
> 
> You can merge this Pull Request by running:
> 
>   git pull https://github.com/afrojer/zsh-config master
> 
> Or you can view, comment on it, or merge it online at:
> 
>   https://github.com/nviennot/zsh-config/pull/1
> 
> -- Commit Summary --
> - added some flexibility to the environment
> 
> -- File Changes --
> 
> M README.md (4)
> A env.d/alias (17)
> A env.d/colors.sh (35)
> A env.d/find_utils.sh (89)
> A gdircolors (194)
> A themes/jeremya.zsh-theme (14)
> M zshrc (55)
> 
> -- Patch Links --
> 
>   https://github.com/nviennot/zsh-config/pull/1.patch
>   https://github.com/nviennot/zsh-config/pull/1.diff

afrojer committed:
Look in custom env dir for color file

nviennot committed:
nog/nogit disable git auto completion

chazy committed:
Updated zsh-syntax highlighting git url

nviennot committed:
Merge pull request #2 from chazy/master

the syntax url path has changed

nviennot committed:
Merge remote-tracking branch 'main/master'

nviennot committed:
Added heroku and bundler plugins

afrojer committed:
Merge remote-tracking branch 'nico/master'

afrojer committed:
Adding default path for port on OSX

afrojer committed:
Really handy gi -> git function for my fat fingers

afrojer committed:
Explicitly use HOME variable not '~' (and export it)

afrojer committed:
nog -> just comdef -d, don't disable git prompt

afrojer committed:
Enable tmux as default shell when USE_TMUX var is set

The USE_TMUX environment variable can be set in a ~/.env.d file.

Signed-off-by: Jeremy C. Andrus <jeremya@cs.columbia.edu>

afrojer committed:
check for 256color-bce AND 256color when exec'ing tmux

Signed-off-by: Jeremy C. Andrus <jeremya@cs.columbia.edu>

afrojer committed:
by default don't start tmux from SSH connections

Signed-off-by: Jeremy C. Andrus <jeremya@cs.columbia.edu>

afrojer committed:
alias tmux to "tmux -2"

Signed-off-by: Jeremy C. Andrus <jeremya@cs.columbia.edu>

afrojer committed:
dircolors: remove CAPABILITY line

This interacted in some way with certain shell scripts which would produce the line:
        Unknown colorls variable `ca'.

Signed-off-by: Jeremy C. Andrus <jeremya@cs.columbia.edu>

afrojer committed:
updates to find utilities

use straight grep, not find piped to xargs

Signed-off-by: Jeremy C. Andrus <jeremy_andrus@apple.com>

afrojer committed:
fixup to find_utils

Signed-off-by: Jeremy C. Andrus <jeremy_andrus@apple.com>

afrojer committed:
update git plugin with zsh/bash completion scripts

Signed-off-by: Jeremy C. Andrus <jeremy_andrus@apple.com>

afrojer committed:
make lol plugin osx friendly

Signed-off-by: Jeremy C. Andrus <jeremy_andrus@apple.com>

afrojer committed:
update the zsh plugins I use

Signed-off-by: Jeremy C. Andrus <jeremy_andrus@apple.com>

afrojer committed:
updates to vi-mode plugin

Signed-off-by: Jeremy C. Andrus <jeremy_andrus@apple.com>

afrojer committed:
Updated zshrc

Signed-off-by: Jeremy C. Andrus <jeremy_andrus@apple.com>

afrojer committed:
Add opp plugin for enhanced vi-mode

Signed-off-by: Jeremy C. Andrus <jeremy_andrus@apple.com>

afrojer committed:
fixup zsh-syntax-highlighting submodule

Signed-off-by: Jeremy C. Andrus <jeremy_andrus@apple.com>

afrojer committed:
don't use vi-mode for now

Signed-off-by: Jeremy C. Andrus <jeremy_andrus@apple.com>

afrojer committed:
add some cool color display functions

This makes it a lot easier to remember how to colorize :-)

Signed-off-by: Jeremy C. Andrus <jeremy_andrus@apple.com>

afrojer committed:
update DIR and LINK colors

Signed-off-by: Jeremy C. Andrus <jeremy_andrus@apple.com>

afrojer committed:
bugfix! preexec can't handle printf fmts

Credit for the fix:
    https://github.com/robbyrussell/oh-my-zsh/issues/890

Signed-off-by: Jeremy C. Andrus <jeremy_andrus@apple.com>

afrojer committed:
add zsh git completion by default

afrojer committed:
Don't sent homebrew analytics

afrojer committed:
Add default macOS MacVim.app path

Signed-off-by: Jeremy C. Andrus <jchristian.andrus@gmail.com>

afrojer committed:
default tmux alias to UTF-8 terminal support

Signed-off-by: Jeremy C. Andrus <jchristian.andrus@gmail.com>

afrojer committed:
Merge remote-tracking branch 'github/master'

Merged 5667161d49b9ddc4ea8de7a379d50fc2cb7ffb50
From: https://github.com/robbyrussell/oh-my-zsh

Signed-off-by: Jeremy C. Andrus <jchristian.andrus@gmail.com>

afrojer committed:
fix the README.md

Signed-off-by: Jeremy C. Andrus <jchristian.andrus@gmail.com>

afrojer committed:
git: use dirty status on empty/spaces-only status

Signed-off-by: Jeremy C. Andrus <jchristian.andrus@gmail.com>

afrojer committed:
updates to jeremya theme: cooler git status chars

Signed-off-by: Jeremy C. Andrus <jchristian.andrus@gmail.com>

afrojer committed:
jeremya: add AHEAD/BEHIND/DIVERGED git prompt status

Signed-off-by: Jeremy C. Andrus <jchristian.andrus@gmail.com>

afrojer committed:
add default PAGER env var

Make sure to set a default pager to avoid screen clearing on man page
viewing and git commands!

Signed-off-by: Jeremy C. Andrus <jchristian.andrus@gmail.com>

afrojer committed:
remove MacVim.app from path
